src/app/components/MainMenu.js

```diff
@@ -0,0 +1,77 @@
+import createVisualImage from './VisualImage';
+import createLogo from './Logo';
+import createButtonRed from './ButtonRed';
+import createWhiteButtonWithIcon from './ButtonWhiteWithIcon';
+import createGameModeName from './GameModeName';
+import ModeMenu, { menuOption } from './ModeMenu';
+import createModeRules from './ModeRules';
+
+// Creating page content
+function PageContent(optionMode = { selectOption: 'People' }) {
```

Comment:
is the fn argument supposed to contain any other data than `selectOption`?
If so, please use es6 destructuring, if not  simplify it setting a string instead

Reply:
Changed for string